assets/javascripts/discourse/connectors/user-card-additional-controls/portfolio-button.hbs

```diff
@@ -0,0 +1,4 @@
+{{#link-to 'userActivity.portfolio' user}}
```

Comment:
@merefield Likewise, this needs to be wrapped in a site setting conditional

Reply:
Done pending commit